cypress/integration/homepage.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Login flow', () => {
+  const CMS_BASEURL = Cypress.env('BASEURL')
+  const COOKIE_NAME = Cypress.env('COOKIE_NAME')
```

Comment:
nit: unused variable. 

As a suggestion (for a separate issue), the first test is failing right now (correctly) because visiting `/` pings the `whoami` endpoint, which sends a 401 since there's no cookie saved. Since we don't handle the 401, this throws an error and fails the test. As a workaround, we can handle the 401 in a try-catch to prevent this test from failing.

Reply:
Good catch on the 401 error - this should be fixed on the routing front (i.e. the LoginContext should not throw a 401 error on the frontend if the user is not logged in on the homepage - because that is the expected behavior anyway). I'll make a separate PR for the fix and rebase this testing PR on top of that.

Reply:
So I took a closer look at this problem.

A possible solution is to get the backend to return `200` for all `whoami` calls
- with no cookie: `userId` value of `null`
- with incorrect cookie: `userId` value of `null`
- with correct cookie: `userId` value of `<USER_ID>`

I'll create a separate issue and PR. Will merge this in the meantime.

Reply:
Created issue https://github.com/isomerpages/isomercms-backend/issues/170